### ISSUE TITLE
Supply Mule does not work with a pure warehouse (without production modules)

### ISSUE THREAD

**Issue by morgenrot68** [Misunderstood-Wookiee/Mules-and-Warehosues-Extended#98]
Supply Mule does not work with a pure warehouse (without production modules)

The following situation:
I wanted to fill my warehouse by the surrounding NPC stations. But this is not possible. 

These are my settings:
Supply Mule:
Home Station: Warehouse
Serve Source Only: activated

Everything below enabled, except "Allow Build Storage"
No goods selected.

The warehouse has no production modules.
Result: Search for trade offers.

I thought it was due to the update to 1.6.4. So I installed the old version again. But the same result.

I still have a shipyard in this sector. This shipyard is supplied immediately, although I have activated "Serve Source Only". This is ignored.The warehouse is completely ignored. Buy orders in the warehouse are available. 
I have built a production module for testing on another warehouse in another sector. This works immediately. 

But as soon as I ONLY have a warehouse without production modules, no trade orders are found. 
As it looks like Supply Mule does not work with a 
pure warehouse. But it would have to be.

**Comment by bhayden53** [comment]
Does the warehouse have a manager and significant budget?

**Comment by morgenrot68** [comment]
Yes.

**Comment by bhayden53** [comment]
Ah sorry, forgot one other thing, on a warehouse you need something to add wares as something they actually are looking to buy. They don't just look to buy everything by default. One way is a station mule with a factory as the source and the warehouse as target, with "make target warehouse" checked. I think there's also a way to manually add wares through either logical overview or by manually making a trade order (then you can give the order back to the manager).

Basically you need the wares to show up in logical overview for any orders to be created by the manager. There used to be an egosoft bug where wares with 0 storage NEVER showed up in logical overview; not sure if that's fixed.

**Comment by bhayden53** [comment]
also worth mentioning is once the station mule has added the wares to the warehouse, he can be re-assigned.

Basically he will take everything from logical overview at the source, and add it to logical overview on the target, when "make target warehouse" is checked. After that, he doesn't need to be a station mule anymore. It's a convenience (in fact when the author originally wrote it, I think it was the only way to add tradewares to a station with no production)

**Comment by morgenrot68** [comment]
Ok, all the goods to be refilled are available in the warehouse at a small quantity. I'll test this immediately with station mule.

**Comment by morgenrot68** [comment]
Perhaps this is the cause.

**Comment by bhayden53** [comment]
If it doesn't work, please post screenshots of the logical overview of the warehouse, the "trade offers" menu showing buy orders, and the supply mule options on the ship's order tab.

**Comment by bhayden53** [comment]
(also, double check your trade rules)

**Comment by morgenrot68** [comment]
unfortunately does not work.
No Trade Rules active.
Supply Mule shows me the goods from the shipyard in the sector and not from the warehouse.

**Comment by bhayden53** [comment]
Get me those screenshots if you can.

Also, do you know how to turn debug on? I could use the debug log for the supply mule assigned to the warehouse.

**Comment by morgenrot68** [comment]
I have activated station mule:

Source: PHQ
Target: Warehouse
everything unchecked except Make Target Warehouse
Does a station mule trade have to be completed first or is one-time activation sufficient?

Debug ? how do I activate it ? In the game menu settings?

Screenes, it takes time.

**Comment by bhayden53** [comment]
I don't think the trade has to be completed, but it'd be good to let him run long enough to make a trade or get stuck in searching for trades. If you have small amounts of goods in the warehouse then that should be sufficient, so I don't think that's going to fix the issue.

The first thing to debug is why no trades are being found for the warehouse; the second is to figure out what's going on with the "Serve source only" that you're seeing. I'll probably need the debug log for that.

To turn debug on, I use this script to launch the game:
https://gist.github.com/MattMcFarland/4c2f5cdd4657e91239cf7285b5d3b726

Make sure to edit the line in the comment:
start "" %X4_EXE_PATH% -showfps -debug all -logfile %LOG_FILE_NAME% -scriptlogfiles %SCRIPT_LOG_FILE_NAME%

Then, when you launch the game, there will be log files created in:
Documents\Egosoft\X4\<number>\

And mule specific logs at
Documents\Egosoft\X4\<number>\logs\mulesextended

In that second folder will be a file for each mule. After launching with debug on, delete the order and restart it, then send that file. I need to see what trades the mule is finding to figure out where to go from here.

**Comment by morgenrot68** [comment]
Here is the log :




ScriptStart
Ship: Sonra Verteidiger (OVS-652) - 0x51abe
Entity (this): 0x51ad3
Supply Mule Settings:
    Home: Argon Prime Handelsstation I (THU-676)
    assignHome: 0
    maxDist: 30
    dedicatedServe: 1
    allowPlayerSuppliers: 1
    allowAiSuppliers: 1
    allowBuildstorage: 0
    allowResources: 1
    allowIntermediates: 1
    allowTradewares: 1
    lockWares: 0
    specialWareBasket: []
    maxTrades: 5
    playerBuyMod: 100
Serving Station or Ship: Argon Prime Handelsstation I
Serving Sector: Argon Prime
Serving Factions: [faction.player]
Current priority is to supply "stationOrShip" by buying "resources" from "player"
------------------------------------------------------------------------------------------------------------------------------------
Current priority is to supply "stationOrShip" by buying "resources" from "ai"
------------------------------------------------------------------------------------------------------------------------------------
Current priority is to supply "stationOrShip" by buying "intermediates" from "player"
------------------------------------------------------------------------------------------------------------------------------------
Current priority is to supply "stationOrShip" by buying "intermediates" from "ai"
------------------------------------------------------------------------------------------------------------------------------------
Current priority is to supply "stationOrShip" by buying "tradewares" from "player"
------------------------------------------------------------------------------------------------------------------------------------
Current priority is to supply "stationOrShip" by buying "tradewares" from "ai"
------------------------------------------------------------------------------------------------------------------------------------
Current priority is to supply "sector" by buying "resources" from "player"
------------------------------------------------------------------------------------------------------------------------------------
----- needs (8):
Argon Prime Schiffswerft I (EJJ-566) wants to buy 1331 Elektronikkomponenten for 131800 (1)
Argon Prime Schiffswerft I (EJJ-566) wants to buy 4551 Energiezellen for 2200 (1)
Argon Prime Schiffswerft I (EJJ-566) wants to buy 662 Feldspulen for 57600 (1)
Argon Prime Schiffswerft I (EJJ-566) wants to buy 315 Nahrungsrationen for 2900 (1)
Argon Prime Schiffswerft I (EJJ-566) wants to buy 219 Hüllenteile for 27200 (1)
Argon Prime Schiffswerft I (EJJ-566) wants to buy 188 Medikamente for 8900 (1)
Argon Prime Schiffswerft I (EJJ-566) wants to buy 402 Schildkomponenten for 26400 (1)
Argon Prime Schiffswerft I (EJJ-566) wants to buy 2233 Geschützkomp. for 38300 (1)

----- supply offers (4):
Profitabler Handel I Hauptquartier (XOD-708) wants to sell 2971 Feldspulen for 43598 (0.14622)
Wiedergutmachung des Kardinals Hüllenteilefabrik III (WKW-932) wants to sell 125000 Energiezellen for 1000 (-1)
Wiedergutmachung des Kardinals Hüllenteilefabrik III (WKW-932) wants to sell 1133 Feldspulen for 24700 (-1)
Wiedergutmachung des Kardinals Hüllenteilefabrik III (WKW-932) wants to sell 12500 Schildkomponenten for 11300 (-1)

ship capacity: 54000
Occupied Cargo: 0
----- trade offer comparisons
Energiezellen target desired amount 136848 supply amount: 54000 ship can hold: 54000 amount will be: 54000 need unitprice: 2200 supp unitprice: 1000 faction buy modifier: 100 profit: 648.000,00 Cr
Feldspulen target desired amount 19860 supply amount: 2971 ship can hold: 3600 amount will be: 2971 need unitprice: 57600 supp unitprice: 43598 faction buy modifier: 100 profit: 504.072,00 Cr
Feldspulen target desired amount 19860 supply amount: 1133 ship can hold: 3600 amount will be: 1133 need unitprice: 57600 supp unitprice: 24700 faction buy modifier: 100 profit: 1.184.400,00 Cr
Schildkomponenten target desired amount 12110 supply amount: 5400 ship can hold: 5400 amount will be: 5400 need unitprice: 26400 supp unitprice: 11300 faction buy modifier: 100 profit: 815.400,00 Cr
Suggested Trade is: Buy 1133 Feldspulen from Wiedergutmachung des Kardinals Hüllenteilefabrik III (WKW-932) and sell to Argon Prime Schiffswerft I (EJJ-566) with 1.184.400,00 Cr of profit.
buying 1133 Feldspulen from Wiedergutmachung des Kardinals Hüllenteilefabrik III (WKW-932) at 247 Cr  to sell to Argon Prime Schiffswerft I (EJJ-566) at 576 Cr for a profit of
Occupied Cargo: 16995
----- trade offer comparisons
Energiezellen target desired amount 136848 supply amount: 54000 ship can hold: 37005 amount will be: 37005 need unitprice: 2200 supp unitprice: 1000 faction buy modifier: 100 profit: 444.060,00 Cr

**Comment by bhayden53** [comment]
Not sure what's going on with "Serve source only" yet but right away it looks like it's not finding any "needs" (i.e. buy orders) for the station you've assigned it to. They're all for the other station.

**Comment by bhayden53** [comment]
If we can't figure out any reason why the warehouse doesn't have buy orders, I'll have to try to see if I can find a way to reproduce. That would have to wait a few hours.

**Comment by morgenrot68** [comment]
ok, in the warehouse buy offers are created.
When I use the other warehouse, where I have added a production module for testing, it works immediately. But as soon as there is no production module, it doesn't work.

**Comment by morgenrot68** [comment]
if it helps, I can also send you the log of it.

**Comment by bhayden53** [comment]
I have a whole bunch of supply mules on a warehouse with no production modules and they're working fine.

Hoe much money does the warehouse have? How many buy orders are there?

**Comment by morgenrot68** [comment]
100 million cr
14 buy offers

**Comment by bhayden53** [comment]
Ok, that's plenty.

I think you have found a bug, we will have to dig in and look for it later. Sorry for the trouble.

**Comment by morgenrot68** [comment]
ok, I guess so.
Let's hope it's as easy to find as the bug yesterday.

**Comment by Galaxy-Spam** [comment]
Hey morgenrot 68,
could you send me a savefile? That would speed stuff up considerably for me.
You can upload it here, or (more privately) send it to me via discord (Galaxy-Spam#1117).

**Comment by morgenrot68** [comment]
Here is the log of the warehouse with production:


ScriptStart
Ship: Sonra Verteidiger (OVS-652) - 0x51abe
Entity (this): 0x51ad3
Supply Mule Settings:
    Home: Wiedergutmachung des Kardinals Hüllenteilefabrik III (WKW-932)
    assignHome: 0
    maxDist: 30
    dedicatedServe: 1
    allowPlayerSuppliers: 1
    allowAiSuppliers: 1
    allowBuildstorage: 0
    allowResources: 1
    allowIntermediates: 1
    allowTradewares: 1
    lockWares: 0
    specialWareBasket: []
    maxTrades: 5
    playerBuyMod: 100
Serving Station or Ship: Wiedergutmachung des Kardinals Hüllenteilefabrik III
Serving Sector: Wiedergutmachung des Kardinals
Serving Factions: [faction.player]
Current priority is to supply "stationOrShip" by buying "resources" from "player"
------------------------------------------------------------------------------------------------------------------------------------
----- needs (2):
Wiedergutmachung des Kardinals Hüllenteilefabrik III (WKW-932) wants to buy 3886 Graphen for 23300 (1)
Wiedergutmachung des Kardinals Hüllenteilefabrik III (WKW-932) wants to buy 18750 Veredelte Metalle for 18435 (0.616102)

----- supply offers (2):
Argon Prime Handelsstation I (THU-676) wants to sell 421 Graphen for 18466 (0.278507)
Argon Prime Handelsstation I (THU-676) wants to sell 876 Veredelte Metalle for 18437 (0.616441)

ship capacity: 54000
Occupied Cargo: 0
----- trade offer comparisons
Veredelte Metalle target desired amount 18750 supply amount: 876 ship can hold: 3857 amount will be: 876 need unitprice: 18435 supp unitprice: 18437 faction buy modifier: 100 profit: -77,14 Cr
no valid trade found
Current priority is to supply "stationOrShip" by buying "resources" from "ai"
------------------------------------------------------------------------------------------------------------------------------------
----- needs (2):
Wiedergutmachung des Kardinals Hüllenteilefabrik III (WKW-932) wants to buy 3886 Graphen for 23300 (1)
Wiedergutmachung des Kardinals Hüllenteilefabrik III (WKW-932) wants to buy 18750 Veredelte Metalle for 18435 (0.616102)

----- supply offers (33):
HAT Erzraffinerie I (RIV-709) wants to sell 409 Veredelte Metalle for 20479 (0.962542)
TEL Komplex für Veredelte Güter I (FYY-208) wants to sell 3987 Graphen for 18468 (0.278806)
PAR High-Tech-Fabrik I (XVW-071) wants to sell 272 Graphen for 20694 (0.611045)
HOP Schiffstechnologie-Fabrik II (BGR-049) wants to sell 8310 Veredelte Metalle for 13297 (-0.254746)
HOP High-Tech-Fabrik VI (XOY-912) wants to sell 4055 Graphen for 17917 (0.196567)
HOP High-Tech-Fabrik II (HTY-640) wants to sell 8130 Veredelte Metalle for 12983 (-0.307966)
ANT High-Tech-Fabrik I (CPV-192) wants to sell 2980 Veredelte Metalle for 15875 (0.182203)
ANT High-Tech-Fabrik VII (LOF-954) wants to sell 2487 Veredelte Metalle for 15470 (0.113559)
ANT Graphen-Raffinerie I (OXW-527) wants to sell 8477 Graphen for 17708 (0.165373)
ARG Komplex für Veredelte Güter I (XFY-717) wants to sell 830 Graphen for 22743 (0.916866)
ANT High-Tech-Fabrik II (RWB-259) wants to sell 115 Graphen for 14613 (-0.301061)
ANT Erzraffinerie I (ORR-062) wants to sell 17071 Veredelte Metalle for 14710 (-0.0152543)
ANT High-Tech-Fabrik IV (BQF-773) wants to sell 1372 Graphen for 18364 (0.263284)
TEL High-Tech-Fabrik II (NEZ-312) wants to sell 2622 Graphen for 17946 (0.200896)
HOP Graphen-Raffinerie I (EBU-523) wants to sell 5678 Graphen for 18529 (0.28791)
HOP Erzraffinerie III (NSB-130) wants to sell 16163 Veredelte Metalle for 13113 (-0.285932)
HOP Erzraffinerie I (XSS-131) wants to sell 20441 Veredelte Metalle for 13052 (-0.296271)
HOP Graphen-Raffinerie II (ZGS-360) wants to sell 6687 Graphen for 19389 (0.416269)
FRF High-Tech-Fabrik I (IVC-652) wants to sell 8010 Veredelte Metalle for 15383 (0.0988136)
ARG High-Tech-Fabrik I (DIY-733) wants to sell 65 Graphen for 18041 (0.215075)
HOP High-Tech-Fabrik II (JOH-915) wants to sell 4499 Graphen for 19097 (0.372687)
HOP Erzraffinerie I (PMB-431) wants to sell 1832 Veredelte Metalle for 18284 (0.590508)
HOP Erzraffinerie II (ERE-214) wants to sell 156 Veredelte Metalle for 20642 (0.990169)
HOP Erzraffinerie III (RGY-084) wants to sell 7456 Veredelte Metalle for 13749 (-0.178136)
PAR Erzraffinerie I (JBN-777) wants to sell 10648 Veredelte Metalle for 16476 (0.284068)
HOP Erzraffinerie I (SED-914) wants to sell 13265 Veredelte Metalle for 14838 (0.00644068)
ARG Komplex für Veredelte Güter I (RJZ-209) wants to sell 5720 Graphen for 15990 (-0.0924242)
ARG Graphen-Raffinerie I (MUC-570) wants to sell 3355 Graphen for 19353 (0.410896)
ARG Erzraffinerie I (VQN-599) wants to sell 10303 Veredelte Metalle for 16727 (0.32661)
ARG High-Tech-Fabrik I (EAS-316) wants to sell 3575 Graphen for 17710 (0.165672)
ARG Erzraffinerie I (QLV-038) wants to sell 13530 Veredelte Metalle for 16457 (0.280847)
HAT Komplex für Veredelte Güter I (ODW-600) wants to sell 6893 Veredelte Metalle for 17383 (0.437797)
ZYA Erzraffinerie II (KUC-061) wants to sell 4996 Veredelte Metalle for 16771 (0.334068)

ship capacity: 54000
Occupied Cargo: 0
----- trade offer comparisons
Graphen target desired amount 3886 supply amount: 2700 ship can hold: 2700 amount will be: 2700 need unitprice: 23300 supp unitprice: 18468 faction buy modifier: 100 profit: 130.464,00 Cr
Graphen target desired amount 3886 supply amount: 2700 ship can hold: 2700 amount will be: 2700 need unitprice: 23300 supp unitprice: 17917 faction buy modifier: 100 profit: 145.341,00 Cr
Graphen target desired amount 3886 supply amount: 2700 ship can hold: 2700 amount will be: 2700 need unitprice: 23300 supp unitprice: 17708 faction buy modifier: 100 profit: 150.984,00 Cr
Graphen target desired amount 3886 supply amount: 830 ship can hold: 2700 amount will be: 830 need unitprice: 23300 supp unitprice: 22743 faction buy modifier: 100 profit: 15.039,00 Cr
Graphen target desired amount 3886 supply amount: 1372 ship can hold: 2700 amount will be: 1372 need unitprice: 23300 supp unitprice: 18364 faction buy modifier: 100 profit: 133.272,00 Cr
Graphen target desired amount 3886 supply amount: 2622 ship can hold: 2700 amount will be: 2622 need unitprice: 23300 supp unitprice: 17946 faction buy modifier: 100 profit: 144.558,00 Cr
Graphen target desired amount 3886 supply amount: 2700 ship can hold: 2700 amount will be: 2700 need unitprice: 23300 supp unitprice: 18529 faction buy modifier: 100 profit: 128.817,00 Cr
Graphen target desired amount 3886 supply amount: 2700 ship can hold: 2700 amount will be: 2700 need unitprice: 23300 supp unitprice: 19389 faction buy modifier: 100 profit: 105.597,00 Cr
Graphen target desired amount 3886 supply amount: 2700 ship can hold: 2700 amount will be: 2700 need unitprice: 23300 supp unitprice: 19097 faction buy modifier: 100 profit: 113.481,00 Cr
Graphen target desired amount 3886 supply amount: 2700 ship can hold: 2700 amount will be: 2700 need unitprice: 23300 supp unitprice: 15990 faction buy modifier: 100 profit: 197.370,00 Cr
Graphen target desired amount 3886 supply amount: 2700 ship can hold: 2700 amount will be: 2700 need unitprice: 23300 supp unitprice: 19353 faction buy modifier: 100 profit: 106.569,00 Cr
Graphen target desired amount 3886 supply amount: 2700 ship can hold: 2700 amount will be: 2700 need unitprice: 23300 supp unitprice: 17710 faction buy modifier: 100 profit: 150.930,00 Cr
Veredelte Metalle target desired amount 18750 supply amount: 3857 ship can hold: 3857 amount will be: 3857 need unitprice: 18435 supp unitprice: 13297 faction buy modifier: 100 profit: 198.172,66 Cr
Veredelte Metalle target desired amount 18750 supply amount: 3857 ship can hold: 3857 amount will be: 3857 need unitprice: 18435 supp unitprice: 12983 faction buy modifier: 100 profit: 210.283,64 Cr
Veredelte Metalle target desired amount 18750 supply amount: 2980 ship can hold: 3857 amount will be: 2980 need unitprice: 18435 supp unitprice: 15875 faction buy modifier: 100 profit: 98.739,20 Cr
Veredelte Metalle target desired amount 18750 supply amount: 2487 ship can hold: 3857 amount will be: 2487 need unitprice: 18435 supp unitprice: 15470 faction buy modifier: 100 profit: 114.360,05 Cr
Veredelte Metalle target desired amount 18750 supply amount: 3857 ship can hold: 3857 amount will be: 3857 need unitprice: 18435 supp unitprice: 15008 faction buy modifier: 100 profit: 132.179,39 Cr
Veredelte Metalle target desired amount 18750 supply amount: 3857 ship can hold: 3857 amount will be: 3857 need unitprice: 18435 supp unitprice: 13113 faction buy modifier: 100 profit: 205.269,54 Cr
Veredelte Metalle target desired amount 18750 supply amount: 3857 ship can hold: 3857 amount will be: 3857 need unitprice: 18435 supp unitprice: 13052 faction buy modifier: 100 profit: 207.622,31 Cr
Veredelte Metalle target desired amount 18750 supply amount: 3857 ship can hold: 3857 amount will be: 3857 need unitprice: 18435 supp unitprice: 15383 faction buy modifier: 100 profit: 117.715,64 Cr
Veredelte Metalle target desired amount 18750 supply amount: 1832 ship can hold: 3857 amount will be: 1832 need unitprice: 18435 supp unitprice: 18284 faction buy modifier: 100 profit: 5.824,07 Cr
Veredelte Metalle target desired amount 18750 supply amount: 3857 ship can hold: 3857 amount will be: 3857 need unitprice: 18435 supp unitprice: 13749 faction buy modifier: 100 profit: 180.739,02 Cr
Veredelte Metalle target desired amount 18750 supply amount: 3857 ship can hold: 3857 amount will be: 3857 need unitprice: 18435 supp unitprice: 16476 faction buy modifier: 100 profit: 75.558,63 Cr
Veredelte Metalle target desired amount 18750 supply amount: 3857 ship can hold: 3857 amount will be: 3857 need unitprice: 18435 supp unitprice: 14838 faction buy modifier: 100 profit: 138.736,29 Cr
Veredelte Metalle target desired amount 18750 supply amount: 3857 ship can hold: 3857 amount will be: 3857 need unitprice: 18435 supp unitprice: 16727 faction buy modifier: 100 profit: 65.877,56 Cr
Veredelte Metalle target desired amount 18750 supply amount: 3857 ship can hold: 3857 amount will be: 3857 need unitprice: 18435 supp unitprice: 16457 faction buy modifier: 100 profit: 76.291,46 Cr
Veredelte Metalle target desired amount 18750 supply amount: 3857 ship can hold: 3857 amount will be: 3857 need unitprice: 18435 supp unitprice: 17383 faction buy modifier: 100 profit: 40.575,64 Cr
Veredelte Metalle target desired amount 18750 supply amount: 3857 ship can hold: 3857 amount will be: 3857 need unitprice: 18435 supp unitprice: 16771 faction buy modifier: 100 profit: 64.180,48 Cr
Suggested Trade is: Buy 3857 Veredelte Metalle from HOP High-Tech-Fabrik II (HTY-640) and sell to Wiedergutmachung des Kardinals Hüllenteilefabrik III (WKW-932) with 210.283,64 Cr of profit.
buying 3857 Veredelte Metalle from HOP High-Tech-Fabrik II (HTY-640) at 129 Cr  to sell to Wiedergutmachung des Kardinals Hüllenteilefabrik III (WKW-932) at 184 Cr for a profit of
Occupied Cargo: 53998
Cargo Full, skip to end
******* went to end

**Comment by morgenrot68** [comment]
oh, sorry, red to late.

**Comment by morgenrot68** [comment]
yes, i can do that, but i have a lot of other mods installed Is that useful?

**Comment by Galaxy-Spam** [comment]
I dont mind installing them, might even reveal other issues.
Just Zip the whole modfolder as well. Unless its several GBs it should be fine.

**Comment by morgenrot68** [comment]
i could try to reproduce it in a mod free installation. But then I would have to build warehouses first. In the save I am just at the beginning, without much credits and ships. So this may take some time.

**Comment by morgenrot68** [comment]
mod folder is 3.3 gb

**Comment by Galaxy-Spam** [comment]
For me thats fine, if you have the bandwidth to upload it somewhere

**Comment by morgenrot68** [comment]
3.3 gb. hmmm, difficult. 
i try to send a save from the mod free installation. But that will take some time. I will not be able to do that today. I have to create the infrastructure in the game. Maybe you can have a look at the code until then.

**Comment by Galaxy-Spam** [comment]
You could take a shortcut with the cheat menu (mod). Anyway, let me first take a look then, no need to go though all this trouble yet.

**Comment by morgenrot68** [comment]
ok, cheat menu is installed. but then i don't have a mod free version. ok, this is what i do.

**Comment by Galaxy-Spam** [comment]
ok, i think i was able to reproduce the part with no tradeoffers found. Should hopefully be an easy fix.

**Comment by morgenrot68** [comment]
I have now tested it on a clean installation, it doesn't work there either, so it really should be a bug.

**Comment by Galaxy-Spam** [comment]
I found the bug, it should be an easy fix. I thought i fixed that a couple of releases ago, but it seems it sneaked back in...

**Comment by morgenrot68** [comment]
But I am glad that it is not a big deal.

**Comment by bhayden53** [comment]
Did we have a merge issue that caused it to get back in?

**Comment by Galaxy-Spam** [comment]
I'm not sure, dev branch looks fine, could have been some merge issue back in the rework branch. Pushing update now.

**Comment by Galaxy-Spam** [comment]
Should be fixed with the next update.
